### ISSUE TITLE
508-defect-3 [SCREENREADER]: Accordion titles should be headings 

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#27615]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I expect titles on the page to be written as headings so that they can be more easily accessible and navigable to me.

## Details

Headings communicate the organization of content on a page and provide valuable in page navigation to assistive technology users like screen readers. On most of the language pages, accordions have heading _like_ titles that are not written as headings which makes them absent to screen readers when navigating by headings on the page.

## Acceptance Criteria

- [ ] Accordion titles are written as headings
- [ ] Headings are ordered logically (confirm using headingsMap extension)

## WCAG or Vendor Guidance (optional)

[W3C Headings](https://www.w3.org/WAI/tutorials/page-structure/headings/)
[Accessible aria accordions (Scott o Hara)](https://www.scottohara.me/blog/2017/10/25/accordion-release.html)

## Screenshots or Trace Logs
<img width="961" alt="Screen Shot 2021-07-20 at 12 43 21 PM" src="https://user-images.githubusercontent.com/14154792/126363301-f63a4429-fc77-4486-a2e1-5426a90fc385.png">

**Comment by laflannery** [comment]
This is not an issue - Accordion titles are headings. A couple of things to note:
- Accordions are in the shadow DOM and do not get caught in headingsMap, ANDI, web devloper Chrome plugin, etc. If you want to confirm that these are markup up correctly, you need to inspect the HTML
- The incorrect heading structure of all accordions is being handled separately by the Public Websites team

I will close this in 1 week unless told otherwise
